smsclient_core/models/sms_gateway.py

```diff
@@ -0,0 +1,220 @@
+# coding: utf-8
+# Copyright (C) 2004-2009 Tiny SPRL (<http://tiny.be>).
+# Copyright (C) 2011 SYLEAM (<http://syleam.fr/>)
+# Copyright (C) 2013 Julius Network Solutions SARL <contact@julius.fr>
+# Copyright (C) 2015 Valentin Chemiere <valentin.chemiere@akretion.com>
+# Copyright (C) 2015 Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api, _
+from odoo.exceptions import UserError
+import logging
+_logger = logging.getLogger(__name__)
+
+PRIORITY_LIST = [
+    ('0', '0'),
+    ('1', '1'),
+    ('2', '2'),
+    ('3', '3')
+]
+
+CLASSES_LIST = [
+    ('0', 'Flash'),
+    ('1', 'Phone display'),
+    ('2', 'SIM'),
+    ('3', 'Toolkit')
+]
+
+
+class SMSClient(models.Model):
```

Comment:
Please split models into separate files as outlined [in Contribution docs](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#file-naming)